modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/podcast/EpisodeManagerImpl.kt

```diff
@@ -445,9 +446,12 @@ class EpisodeManagerImpl @Inject constructor(
         }
     }
 
-    override fun starEpisode(episode: PodcastEpisode, starred: Boolean) {
-        episode.isStarred = starred
```

Comment:
I removed this call to `episode.isStarred` because this is already being done in the `MediaSessionManager`. The only other place that calls this method doesn't need to do this because they are just using an episode instance that is being fetched from the db, and not being used after that.

The `MediaSessionManager`, however, is using the episode instance from `PlaybackManager::getCurrentEpisode` which is retrieved from our `UpNextQueue` object. It appears that the `UpNextQueue` does not listen for changes to episodes in the queue, which I'm guessing is the reason we're manually updating it in the `MediaSessionManager` when we change the starred value.

Ideally, we would be listening for changes in the `UpNextQueue`, but until then I left the manual updates to the episode instance in `MediaSessionManager` to keep that object up-to-date. I didn't observe any adverse effects from removing that update in my testing, but it seems better to just keep the current behavior, which is why I only stopped setting the starred value a second time inside the `EpsiodeManagerImpl`. I went ahead and stopped setting it twice since ideally we shouldn't ever be setting it, so avoiding setting it a second time gets us one (very tiny) step closer to that.